### ISSUE TITLE
Problems when adding mutiple nodes to telegram webhook

### ISSUE THREAD

**Issue by JureGrinffin** [xddxdd/bird-lg-go#43]
I've set up my lg-proxy ans lg-frontend using docker-compose, and it works properly.
I've set up reverse proxy using nginx, applied cdn conf in cloudflare.

Now I'm trying to set up my telegram webhook.
It works properly when I added only one node to webhook.
But it gives return below when I was trying to add all my nodes.
```
request failed: invalid server
```
My looking glass is at https://lg.jg-awsl.xyz/
My nodes: dn42-fra4,dn42-nyc4,dn42-sgp4,dn42-ams4
My domain: jg-awsl.xyz
My curl command: 
```
curl "https://api.telegram.org/bot${BotAPIToken}/setWebhook?url=https://lg.jg-awsl.xyz/telegram/dn42-fra4+dn42-nyc4+dn42-sgp4+dn42-ams4"
```

**Comment by JureGrinffin** [comment]
And how to configure dns if I want to use whois command?

**Comment by xddxdd** [comment]
Sorry for replying late, I was busy in the last few weeks.

Your looking glass has an uninitialized buffer bug which was fixed in #41. Basically, this makes the response from `/trace` super long, and the response from the second server is truncated.

Please upgrade the LG to the latest version.

**Comment by JureGrinffin** [comment]
Upgraded, all problems solved.
Thanks!
